packages/tracing/src/browser/metrics.ts

```diff
@@ -134,6 +134,43 @@ export class MetricsInstrumentation {
 
     // Measurements are only available for pageload transactions
     if (transaction.op === 'pageload') {
+      // normalize applicable web vital values to be relative to transaction.startTimestamp
+
+      const timeOrigin = msToSec(performance.timeOrigin);
+
+      ['fcp', 'fp', 'lcp', 'ttfb'].forEach(name => {
+        if (!this._measurements[name] || timeOrigin >= transaction.startTimestamp) {
+          return;
+        }
+
+        // The web vitals, fcp, fp, lcp, and ttfb, all measure relative to timeOrigin.
+        // Unfortunately, timeOrigin is not captured within the transaction span data, so these web vitals will need
+        // to be adjusted to be relative to transaction.startTimestamp.
+
+        const oldValue = this._measurements[name].value;
+        const measurementTimestamp = timeOrigin + msToSec(oldValue);
+        // normalizedValue should be in milliseconds
+        const normalizedValue = (measurementTimestamp - transaction.startTimestamp) * 1000;
+
+        const delta = normalizedValue - oldValue;
+        logger.log(
+          `[Measurements] Normalized ${name} from ${this._measurements[name].value} to ${normalizedValue} (${delta})`,
+        );
+
+        this._measurements[name] = { value: normalizedValue };
```

Comment:
```suggestion
        this._measurements[name].value = normalizedValue;
```

Should this just set the value to the normalized value? This way it will persist other fields that may be added in the future.